src/proxy-ui-api/frontend/src/views/Service/Endpoints/Endpoints.vue

```diff
@@ -77,8 +82,16 @@ export default Vue.extend({
   },
   computed: {
     ...mapGetters(['service']),
+
+    endpoints(): Endpoint[] {
+      const filtered = this.service.endpoints.filter((endpoint: Endpoint) => {
+        return endpoint.method === '*' && endpoint.path === '**';
```

Comment:
Currently endpoints view lists only base endpoint. I think this should be 

`return endpoint.method !== '*' && endpoint.path !== '**';`